docs/ybm_cluster_encryption_update-state.md

```diff
@@ -0,0 +1,37 @@
+## ybm cluster encryption update-state
+
+Update Encryption at Rest (EaR) state for a cluster
+
+### Synopsis
+
+Update Encryption at Rest (EaR) state for a cluster
+
+```
+ybm cluster encryption update-state [flags]
+```
+
+### Options
+
+```
+      --enable   [REQUIRED] Whether to enable or disable EAR (default true)
+  -h, --help     help for update-state
+```
+
+### Options inherited from parent commands
```

Comment:
@cdavid Is it possible to specify this for all commands by default? ie unless overridden these options are always inherited by child commands

Reply:
ie, a common set of overrides in a common place instead of specifying these overrides in every command

Reply:
Does as globals flags which can be overriden for every command, or inherited from the parents command for example cluster-name,  so nothing abnormal in my opinion. The help describe exactly what is possible or not.

Most the CLI use something similar, flag of current command and global flags.  For example Terraform.

If the number of global options start to be large then we should consider `ybm options` like what kubectl is doing., but I think we are still ok. 

wdyt @posriniv @cdavid

Reply:
I agree with @shinji62

Reply:
Ok, makes sense